OpenMEEG/libs/OpenMEEGMaths/include/matrix.h

```diff
@@ -188,8 +188,9 @@ namespace OpenMEEG {
     inline double Matrix::frobenius_norm() const {
     #ifdef HAVE_LAPACK
     if ( nlin()*ncol() != 0 ) {
-        double work;
-        return DLANGE('F',sizet_to_int(nlin()),sizet_to_int(ncol()),data(),sizet_to_int(nlin()),&work);
+        double work = -1.;
```

Comment:
#define UNUSED(expr) do { (void)(expr); } while (0)
Add a call to this macro in BlasLapackImplementations/OpenMEEGMathsOpenBLASConfig.h